lib/gdpr/gateway/session.rb

```diff
@@ -6,15 +6,17 @@ def delete_sessions
 
     logger.info('Starting daily session deletion')
 
-    rows = Session.where(Sequel.lit('start < DATE_SUB(DATE(NOW()), INTERVAL 32 DAY')).count
+    total = 0
+    loop do
+      deleted_rows = DB[:sessions].with_sql_delete('DELETE FROM sessions WHERE start < DATE_SUB(DATE(NOW()), INTERVAL 32 DAY) LIMIT 500')
```

Comment:
tiny suggestion for cleaner code: can we move the magic number 500 into a constant declared at the top?

Reply:
Here we go, change made here: https://github.com/alphagov/govwifi-logging-api/pull/222/commits/8a7ed7e881224c7157b82a99d826baec6cfc4716